src/find.js

```diff
@@ -1,6 +1,7 @@
 const _ = require('underscore');
 const sanitizeParams = require('./utils/sanitizeParams');
 const { prepareResponse, generateSort, generateCursorQuery } = require('./utils/query');
+var config = require('./config');
```

Comment:
I think we generally prefer `const` and `let` over `var`

```suggestion
const config = require('./config');
```